src/librustdoc/html/static/js/main.js

```diff
@@ -784,18 +791,25 @@ function preLoadCss(cssUrl) {
         }
         if (window.CURRENT_TOOLTIP_ELEMENT && window.CURRENT_TOOLTIP_ELEMENT.TOOLTIP_BASE === e) {
             // Make this function idempotent.
+            clearTooltipHoverTimeout(window.CURRENT_TOOLTIP_ELEMENT);
```

Comment:
Nit: The existing comment "Make this function idempotent", which used to be attached to the early return, now looks like it is saying that `clearTooltipHoverTimeout` makes the function idempotent.

Probably a better place for a comment would be above the `if` clause here, saying something like "if this tooltip is already shown, avoid doing additional work."